install/0000_80_machine-config-operator_00_namespace.yaml

```diff
@@ -53,3 +53,13 @@ metadata:
   labels:
     name: openshift-vsphere-infra
     openshift.io/run-level: "1"
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-kubevirt-infra
```

Comment:
we're not going with this - we're decoupling everything not MCO that does leverage the MCO in this way. There are BZs around this as well. We'd need to discuss this further - the full rationale is that we need to find a way for my team to own everything in the MCO repo w/o saying "this isn't our team's stuff". This goes against that

Reply:
It makes a lot of sense to decouple them, and I'll be happy to implement it differently.
Is there already another way to achieve that?
We aim to support the new platform as dev-preview in 4.7, and the KNI-networking is a must for that.
Can you refer me to the work that is done around that, maybe tickets/BZs.
If the issue is mainly creating a new resource (the namespace), I believe we can create it in the platform-specific area in the installer, but adding cases in the existing switch-cases should probably need to be done here.

Reply:
we're unfortunately not there yet as far as a new design/architecture goes - time wise I think landing the templates is ok'ish as long as we decouple the namespace. This is a cross-team discussion that we haven't fully started yet but we will asap

Reply:
Thank you.
I removed the namespace manifest from the 'install' dir

Reply:
The decoupling work sounds like a good approach, and since we are maintaining a platform (KubeVirt) for IPI, I'll be happy to be involved or at least updated.
Are there already any tickets that we can follow?

Reply:
@runcom I tested this branch with an installer that creates the namespace in its manifests, and the installation finished successfully. The cluster is functioning as expected with all the KNI components.
Can we cancel the /hold now?